src/index.js

```diff
@@ -59,18 +60,18 @@ const completeTask = (index) => { // function for change the comple status to co
   addToHTML();
 };
 /* eslint-anable */
-addTasksBtn.addEventListener('click', () => { // by clicking the button
-  tasks.push(new Task(deskTaskInput.value)); // add new task
-  updateLocal(); // save to Local Storage
-  addToHTML(); // call function wich add new task to HTML template
-  deskTaskInput.value = ''; // clear the input after clicking "Add" button
+addTasksBtn.addEventListener('click', () => { 
+  tasks.push(new Task(deskTaskInput.value));
+  updateLocal(); 
+  addToHTML(); 
+  deskTaskInput.value = ''; 
 
 });
 
-const deleteTask = index => { // function for delete button
+const deleteTask = index => {
   todoItemElems[index].classList.add('deletion');
-  setTimeout(() => { // create a pause for animation when delete
-    tasks.splice(index, 1); // splice delete the index
+  setTimeout(() => { 
+    tasks.splice(index, 1);
     updateLocal();
     addToHTML();
   }, 500);
```

Comment:
- [ ] When we delete a task, all indices should be updated and reordered to follow the normal order, and they should be unique
![Screenshot from 2021-08-21 19-01-12](https://user-images.githubusercontent.com/56886064/130329480-8d73c963-a7f4-4be5-afae-853ef17cd44b.png)

Reply:
Fix it